### PR TITLE
doc: Document aliases for command line arguments

### PR DESCRIPTION
**Pull request by pkratoch**
Closes: https://github.com/rpm-software-management/dnf5/issues/131

### CODE REVIEW

**Comment by pkratoch** [comment]
Changed the `named_arg` description, namely fixed the `has_value` part and added example by @jrohel that shows the `${}` substitution.

File: doc/misc/aliases.7.rst [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+        ['download.dest']
+        type = 'named_arg'
+        long_name = 'dest'
+        has_value = true
+        value_help = 'DESTDIR'
+        attached_named_args = [
+            { id_path = 'download.destdir' }
+        ]
+
+  - Alias ``--setflags=TS_FLAGS`` for ``--setopt=tsflags=TS_FLAGS``:
```
Comment (kontura) [inline doc/misc/aliases.7.rst:212]: I think here it should say --set**ts**flags=TS_FLAGS.

File: doc/misc/aliases.7.rst [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+---------------
+
+The ``named_arg`` defines an alias that can replace multiple options and can define a value for each.
+
+Keys:
+    - ``type`` with value ``named_arg``
+    - ``long_name`` - name of the alias option
+    - ``short_name`` - one-letter shortcut of the name
+    - ``descr`` - description that will be shown in help
+    - ``has_value`` - whether the option requires a value. The value is then substituted for ``${}`` strings in the
```
Comment (kontura) [inline doc/misc/aliases.7.rst:173]: Just a suggestion but perhaps we could specify that both `has_value` and `complete` are set to `false` by default?

File: doc/misc/aliases.7.rst [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+      values of ``attached_named_args``.
+    - ``value_help`` - the string shown in help for the value (e.g. ``CONFIG_FILE_PATH`` for
+      ``--config=CONFIG_FILE_PATH``)
+    - ``const_value`` - default constant value (specified only if the alias does not have a value on the command line)
```
Comment (kontura) [inline doc/misc/aliases.7.rst:177]: Maybe we could also mention that the value is substituted for ``${}``?

**Review by kontura — COMMENTED** [review]
This is great!

**Comment by pkratoch** [comment]
I made the suggested changes, thanks. I also made a cosmetic change and altered the keys descriptions to be whole sentences (i.e. start with capital letter, end with dot), to make it look a bit more unified, because some descriptions are longer and required multiple sentences.

**Comment by kontura** [comment]
Thank you, LGTM.

**Pull request merged**